graspit-simulator/graspit
# Cmake install to install graspit globally

configured cmake to install graspit globally. 
copies models, worlds and images into home/$USER/.graspit
And reorganized header files to be imported relative to the parent graspit include folder ex: include graspit/EGPlanner/egPlanner.h 
moved cmdline and ply files to conform with the include directory structure. 
cleaned up the CMakeLists.txt

To test:
```
git clone https://github.com/iretiayo/graspit.git -b cmake_install
mkdir build
cd build
cmake ..
make -j5
sudo make install
```
Now graspit should be installed globally. You can confirm on new terminal:
```
export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
graspit_simulator
```
@mateiciocarlie @jvarley @mkghaas
